common/translation_utils/update_status.py

```diff
@@ -0,0 +1,91 @@
+import csv
+import polib
+
+class Translation:
+    def __init__(self, row):
+        self.phrase = row[3]
+        self.bad_locales = row[1].split(',')
+        self.bad_source_phrase = row[2]
+        self.tag = row[0]
+        self.translations = row[3:]
+
+    def needs_improvement(self):
+        return self.bad_source_phrase != ''
+
+    def is_phrase(row):
+        return row[0] in ['', 'sg', 'pl']
+
+def update_status_run():
+    
+    def lookup_status(msg, replacement_strings=[]):
```

Comment:
It's pretty uncommon to define functions within other functions.
You can just keep them outside of `updated_status_run`.

Reply:
Have removed nested functions